Header header
int32 unique_id
TrackedPointHMM[] lane_keeping_path
TrackedPointHMM[] right_change_path
TrackedPointHMM[] left_change_path